### PR TITLE
DR-967 auto version on develop merge

### PR DESCRIPTION
**Pull request by thathert**

### CODE REVIEW

File: .github/workflows/dev-image-update.yaml [hunk 1]
```diff
@@ -20,22 +20,31 @@ jobs:
...
         with:
           actions_subcommand: 'deploytagupdate'
           helm_env_prefix: dev
       - name: "Check and edit Helm definition for integration"
-        uses: broadinstitute/datarepo-actions@0.6.0
+        uses: broadinstitute/datarepo-actions@0.8.1
         with:
           actions_subcommand: 'deploytagupdate'
-          helm_env_prefix: 'integration-1,integration-2,integration-3,integration-4,integration-5'
+          helm_env_prefix: 'integration-4,integration-5'
```
Comment (snf2ye) [inline .github/workflows/dev-image-update.yaml:47]: Why is this changing to only impact int-4 & 5?

Comment (thathert) [reply .github/workflows/dev-image-update.yaml:47]: We shouldn't be auto-deploying to int-1, int-2, or int-3 when we merge a branch. It means that any integration tests that might be running against those envs will not be testing the right code any more.

Comment (thathert) [reply .github/workflows/dev-image-update.yaml:47]: We reserve int-4 and int-5 for UI integration tests, and so we want those to always have the latest dev image.

**Review by fboulnois — APPROVED** [review]
thanks for the response, lgtm

**Review by smark88 — APPROVED** [review]

File: .github/workflows/dev-image-update.yaml [hunk 2]
```diff
@@ -20,22 +20,31 @@ jobs:
           repository: 'broadinstitute/datarepo-helm-definitions'
           token: ${{ secrets.HELM_REPO_TOKEN }}
           path: datarepo-helm-definitions
+      - name: "Bump the tag to a new version"
+        uses: broadinstitute/datarepo-actions@0.8.1
```
Comment (smark88) [inline .github/workflows/dev-image-update.yaml]: Next tagged release is gonna be 0.7.0 unless you changed https://github.com/broadinstitute/datarepo-actions/blob/master/.github/workflows/tag.yaml

**Pull request merged**